lib/stdlib/doc/src/maps.xml

```diff
@@ -364,6 +364,26 @@ none</code>
       </desc>
     </func>
 
+    <func>
+      <name name="merge_deep" arity="2" since=""/>
+      <fsummary></fsummary>
+      <desc>
+        <p>Merges two maps, and if two keys exist in both maps
+          and both values are maps, merges them recursively.</p>
+        <p>If either of values at the same key is not map,
```

Comment:
I would probably replace "If either of values at the same key is not map" with "Otherwise".

Reply:
ty fixed